assets/airports/ebbr.json

```diff
@@ -1,241 +1,1872 @@
-                "07L": ["SOUTH"],
-                "07R": ["SOUTH"]
-            },
-            "draw": [["EBBR","SOUTH"]]
-        },
-        "WEST": {
-            "icao": "WEST",
-            "name": "WEST",
-            "rwy": {
-                "19" : ["WEST"],
-                "01" : ["WEST"],
-                "25L": ["WEST"],
-                "25R": ["WEST"],
-                "07L": ["WEST"],
-                "07R": ["WEST"]
-            },
-            "draw": [["EBBR","WEST"]]
+        "LNO": {
+            "icao": "LNO",
+            "name": "Olno",
+            "suffix": {"01":"6F", "07L":"5H", "07R":"5Y", "19":"6L", "25L":"5Q", "25R":"5C"},
+            "rwy": {
+                "01": ["HUL06", ["R045", "A60"]],
+                "07L": ["AFI17", "AFI22", "REMBA"],
+                "07R": ["BUB04", "BUB08", "REMBA"],
+                "19": ["BR010", ["BR011", "A60"]],
+                "25L": [],
+                "25R": []
+            },
+            "exitPoints": {
+                "LNO": [
+                    "LNO"
+                ]
+            },
+            "draw": [
+                ["HUL06", "R045", "LNO"],
+                ["AFI17", "AFI22", "REMBA"],
+                ["BUB04", "BUB08", "REMBA", "LNO"],
+                ["BR010", "BR011", "LNO"]
+            ]
+        },
+        "SPI": {
+            "icao": "SPI",
+            "name": "Sprimont",
+            "suffix": {"01":"6F", "07L":"6H", "07R":"1Y", "19":"5L", "25L":"5Q", "25R":"5C"},
+            "rwy": {
+                "01": ["HUL06", ["R045", "A60"]],
+                "07L": ["AFI17", "AFI22", "REMBA", "BULUX"],
+                "07R": ["HUL06", ["R045", "A60"]],
+                "19": ["BR010", ["BR011", "A60"]],
+                "25L": [],
+                "25R": []
+            },
+            "exitPoints": {
+                "SPI": [
+                    "SPI"
+                ]
+            },
+            "draw": [
+                ["HUL06", "R045", "SPI"],
+                ["AFI17", "AFI22", "REMBA", "BULUX", "SPI"],
+                ["BR010", "BR011", "SPI"]
+            ]
+        },
+        "SOPOK": {
+            "icao": "SOPOK",
+            "name": "Sopok",
+            "suffix": {"01":"6F", "07L":"5H", "07R":"1Y", "19":"6L", "25L":"4D", "25R":"6Z"},
+            "rwy": {
+                "01": ["HUL", "REMBA"],
+                "07L": ["HUL", "REMBA"],
+                "07R": ["HUL", "REMBA"],
+                "19": [["BR012", "A50"], "BR013", "REMBA"],
+                "25L": ["BUB8W"],
+                "25R": ["BUB06", "BUB8W", "REMBA"]
+            },
+            "body": [
+                ["BULUX", "A170"]
+            ],
+            "exitPoints": {
+                "SOPOK": [
+                    "SOPOK"
+                ],
+                "ETENO": [
+                    "SOPOK", ["ETENO", "A250"]
+                ],
+                "LIRSU": [
+                    "SOPOK", ["ETENO", "A250", "LIRSU"]
+                ]
```

Comment:
my personal preference, which is by no means convention and actually a down-right anti-convention, is to go with:
```javascript
"LIRSU": ["SOPOK", ["ETENO", "A250", "LIRSU"]]
```
We have such a custom implementation here, that I think this format makes sense.  Expect in the case of the `draw` section, where it makes sense to break those up on one route per line.
@RNGFavorsNoobs (we should add this info to #184 )